toolium/utils/dataset.py

```diff
@@ -132,7 +133,8 @@ def _replace_param_replacement(param, language):
         '[EMPTY]': '',
         '[B]': ' ',
         # make sure random is not made up of digits only, by forcing the first char to be a letter
-        '[RANDOM]': ''.join([random.choice(string.ascii_lowercase), *(random.choice(alphanums) for i in range(7))]),
+        '[RANDOM]': ''.join([r.choice(string.ascii_lowercase), *(r.choice(alphanums) for i in range(7))]),
+        '[RANDOM_PHONE_NUMBER]': ''.join(['+', '3', '4', '6', '5', '4'] + [str(r.randint(0, 9)) for i in range(1, 7)]),
```

Comment:
why does it have to start with +34654? maybe it should be a more generic solution

Reply:
There are a lot of possible phone number formats depending on the country, on their type (landline, mobile) on their representation scheme (local, national, international). We could add some of these variants on demand by adding suffixes to the default replacement, e.g. RANDOM_PHONE_NUMBER_ES_MOBILE_INT, but I would not tackle that task until proven needed.

For the default replacement, a properly formed Spanish mobile number in international format seems like a good option. We set a fixed prefix for the national part of it (654) thinking that maybe it can be useful to identify these made-up phone numbers in the systems where they are to be used, but it is not really necessary. We can leave just the initial digit (6) and let the rest be chosen at random. What do you think?

Reply:
ok, we can merge it as is and update it later